framework/src/test/java/org/tron/core/net/messagehandler/TransactionsMsgHandlerTest.java

```
@@ -0,0 +1,87 @@
+      Assert.assertFalse(transactionsMsgHandler.isBusy());
+
+      transactionsMsgHandler.init();
+
+      PeerConnection peer = Mockito.mock(PeerConnection.class);
+      TronNetDelegate tronNetDelegate = Mockito.mock(TronNetDelegate.class);
+      AdvService advService = Mockito.mock(AdvService.class);
+
+      Field field = TransactionsMsgHandler.class.getDeclaredField("tronNetDelegate");
+      field.setAccessible(true);
+      field.set(transactionsMsgHandler, tronNetDelegate);
+
+      BalanceContract.TransferContract transferContract = BalanceContract.TransferContract
+          .newBuilder()
+          .setAmount(10)
+          .setOwnerAddress(ByteString.copyFrom(ByteArray.fromHexString("121212a9cf")))
+          .setToAddress(ByteString.copyFrom(ByteArray.fromHexString("232323a9cf"))).build();
+
+      long transactionTimestamp = DateTime.now().minusDays(4).getMillis();
+      Protocol.Transaction trx = Protocol.Transaction.newBuilder().setRawData(
+          Protocol.Transaction.raw.newBuilder().setTimestamp(transactionTimestamp)
+          .setRefBlockNum(1)
+          .addContract(
+              Protocol.Transaction.Contract.newBuilder()
+                  .setType(Protocol.Transaction.Contract.ContractType.TransferContract)
+                  .setParameter(Any.pack(transferContract)).build()).build())
+          .build();
+      Map<Item, Long> advInvRequest = new ConcurrentHashMap<>();
+      Item item = new Item(new TransactionMessage(trx).getMessageId(),
+          Protocol.Inventory.InventoryType.TRX);
+      advInvRequest.put(item, 0L);
+      Mockito.when(peer.getAdvInvRequest()).thenReturn(advInvRequest);
+
+      List<Protocol.Transaction> transactionList = new ArrayList<>();
+      transactionList.add(trx);
+      transactionsMsgHandler.processMessage(peer, new TransactionsMessage(transactionList));
+      Assert.assertNull(advInvRequest.get(item));
+      //Thread.sleep(10);
+    } catch (Exception e) {
+      logger.error("error", e);
```

should use Assert.fail() instead of logger.error?